src/components/Footer.js

```diff
@@ -21,10 +21,16 @@ const Footer = () => {
       </div>
       <div>
         <h5>FOLLOW US</h5>
-        <div>
-          <FaInstagramSquare />
-          <FaTwitterSquare />
-          <FaFacebookSquare />
+        <div className="social-media-icons">
+          <a href="https://www.instagram.com">
```

Comment:
Nice. This works, but let's open the link in a new browser tab. 
Currently it's openning in the same tab, and we're losing out website.

Reply:
okay got that fixed.